# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# T2 SDE: package/.../mine/ncurses.patch
# Copyright (C) 2013 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mine-0.23/Makefile.vanilla	2013-10-10 17:50:18.187617508 +0200
+++ mine-0.23/Makefile	2013-10-10 17:50:37.199616851 +0200
@@ -83,7 +83,7 @@
 	$(CC) -static $(MINE_ALL_OBJS) $(MINE_ALL_LIBS) -o mine.static
 
 gasgui: $(GAS_OBJ)
-	$(CC) $(GAS_OBJ) -ldialog -lcurses -lm -o gasgui
+	$(CC) $(GAS_OBJ) -ldialog -lncursesw -lm -o gasgui
 
 install: all
 	install mine $(sysprefix)/bin/mine
